src/09-workorders/work-orders.apib

```diff
@@ -39,6 +39,14 @@ Get details for a single work order.
             + location (Address)
             + created_at: `2019-05-09T11:25:11+00:00` (string)
             + updated_at: `2019-05-09T11:30:58+00:00` (string)
+            + work_hours (array)
+                + (object)
+                    + user (object)
+                        + type: `user` (string)
+                        + id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string)
+                    + started_at: `2019-02-27T10:01:49+00:00` (string)
+                    + duration: 3600 (number) - In seconds
+                    + description (string)
```

Comment:
Q: no example for the description?

Reply:
The draft endpoint doesn't provide one either. So let's keep it the same since it ought to be showing info for the same one.

Reply:
Would update both of them. These examples/placeholders are helpful for our integrators.